addons/mail/models/res_users_settings_volumes.py

```diff
@@ -8,6 +8,7 @@ class ResUsersSettingsVolumes(models.Model):
     """ Represents the volume of the sound that the user of user_setting_id will receive from partner_id. """
     _name = 'res.users.settings.volumes'
     _description = 'User Settings Volumes'
+    _rec_name = 'partner_id'
```

Comment:
For the other 2 it should be fine as you selected fields that completely identify the records by themselves, but for this one this information will be incomplete, the proper way to identifying records of this model is by the combination of `user_setting_id` and either `partner_id` or `guest_id`.

Maybe override of `name_get` would give better results?

Reply:
hi @seb-odoo, as suggested updated with name_get method for the model res.users.settings.volumes